test/DotNetWorkerTests.Shared/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Microsoft.Azure.Functions.Worker.Tests, PublicKey=00240000048000009400000006020000002400005253413100040000010001005148be37ac1d9f58bd40a2e472c9d380d635b6048278f7d47480b08c928858f0f7fe17a6e4ce98da0e7a7f0b8c308aecd9e9b02d7e9680a5b5b75ac7773cec096fbbc64aebd429e77cb5f89a569a79b28e9c76426783f624b6b70327eb37341eb498a2c3918af97c4860db6cdca4732787150841e395a29cfacb959c1fd971c1")]
```

Comment:
Being a test project which will never be published anywhere, using `public` types is fine. No need for internals - we aren't providing this to customers, the public vs internal rules are much more lax here.

Reply:
Some of the test types use internal types from the worker core package so there's a mismatch exception when making them public. Is there a better way to handle this?

Reply:
Can you give link some example types?

Reply:
[OutputBindingsInfo](https://github.com/Azure/azure-functions-dotnet-worker/blob/main/src/DotNetWorker.Core/OutputBindings/OutputBindingsInfo.cs)  is used in [TestFunctionBindingsFeatures](https://github.com/Azure/azure-functions-dotnet-worker/blob/main/test/DotNetWorkerTests/Features/TestFunctionBindingsFeature.cs). The latter is used in both the otel test project and the main DotNetWorker test project.

Reply:
`OutputBindingsInfo` is not from `DotNetWorkerTests.Shared.csproj`. How are these `InternalsVisibleTo` impacting that?

Reply:
Or is the issue a build error, which I don't remember the exact verbage, but something like "Type X is less visible than type Y".

Reply:
@jviau Yeah the build one which I think you're referring to: _Inconsistent accessibility: property type 'OutputBindingsInfo' is less accessible than property 'TestFunctionBindingsFeature.OutputBindingsInfo'_

Reply:
Ah okay - then InternalsVisisbleTo will be needed.